tests/test_custom_datamodule.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
clear indication, that the package setup is broken. use the python package setup please.